### DISCUSSION TITLE
Where can I find documentation?

### DISCUSSION THREAD

**Discussion by mnemosdev** [maplibre/maplibre-gl-js#69] [General]
Hello,

where can I find docs for this library? Is it the same, as of now, as Mapbox?

Thanks

**Comment by jvolker** [comment]
I'm new to this project and had the same question. 

There is a repo here: https://github.com/maplibre/maplibre-gl-js-docs
And I've found this as well: https://wipfli.github.io/maplibre-gl-js-docs/ Not sure if the latter is up to date though.

**Comment by GoodClover** [comment]
https://maplibre.org/maplibre-gl-js-docs/api/
A lot of the links seem to go to Mapbox still though

**Comment by wipfli** [comment]
Hi. Thanks for asking about the documentation. The official MapLibre documentation is available at https://maplibre.org/maplibre-gl-js-docs/. We are in process of migrating this website from Mapbox to MapLibre so there are still a lot of references to Mapbox as you pointed out correctly.

If you feel like you want to contribute to the docs repo and replace a Mapbox here and there with MapLibre you are more than welcome to fork the documentation repo https://github.com/maplibre/maplibre-gl-js-docs, make some changes to it, and submit a pull request. We are happy about all helping hands.

Feel free to also use the GitHub Issues of https://github.com/maplibre/maplibre-gl-js-docs to suggest improvements.
